V5 docker revised guide, updated community links and resources, added instructions for V6 testnet install for Arch Linux

Valcyclovir opened the pull request:
This is BRX from the community channel reporting in, I am bringing in some changes to Milian's guide on his complete installation guide. This guide is being widely shared among the community and making sure it is updated frequently is crucial ! Thanks.

Valcyclovir commented:
Please review the previous changes whenever possible. This guide by Milian is widely used by new node runners and I will try to keep it updated as we go.

Valcyclovir commented:
Arch Linux guide tested using EndeavourOS

jlsam reviewed (changes requested):
There are many FTP programs that support sftp.
Filezilla is probably the most well known, but I suggest it should be mentioned as an example only.

To use just a server name in Files, the user needs to have a properly set up ~/.ssh/config file. That is beyond the scope of this guide. Perhaps a more 'manual' example like sftp://user@ip would be more clear

Valcyclovir commented:
@jlsam Thanks for the feedback ! Since Filezilla is FOSS, I didn't mind straight up suggesting it but you are right, I changed that part a little. Turns out it's also available for Ubuntu (and other Linux platforms) so it will be the go-to suggestion for anyone struggling to move the file over.

Valcyclovir commented:
Updates today:
- Added blazegraph support for Arch users to reflect most recent updates
- Removed manual installation guide for dockerless since it's outdated (nohub and forever no longer needed), no reasons not to use calvin's one script install
- Updated suggested way to move graphDB to test node server

NZT48 commented on dkg-v6-upcoming-version/setup-instructions-dockerless.md:
````diff
@@ -48,155 +52,23 @@ apt install git -y && cd /root && git clone https://github.com/OriginTrail/ot-no
+systemctl restart otnode
 ```
 
 ****\
````
Please remove this
